src/test/java/co/omise/live/LiveCustomerRequestTest.java

```diff
@@ -0,0 +1,149 @@
+package co.omise.live;
+
+import co.omise.Client;
+import co.omise.models.Customer;
+import co.omise.models.OmiseException;
+import co.omise.models.Ordering;
+import co.omise.models.ScopedList;
+import co.omise.requests.Request;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static junit.framework.TestCase.assertEquals;
```

Comment:
I think we should use `org.junit` in Junit 4 stead of using `junit.framework` in Junit 3.

Reply:
Yeah right, I need to be more careful with my imports 🤦‍♂️